exercises/B-boolean-literals/exercise.js

```diff
@@ -6,6 +6,9 @@
 */
 
 let codeYourFutureIsGreat = true;
+let mozafarIsCool = false;
+let calculationCorrect = true;
+let moreThan10Students = false;
 
```

Comment:
Hi Temwa. This exercise is to demonstrate statements with values being boolean. Good job keeping the spacing consistent before and after the = sign. Clean code is good code.